crates/cxx-qt-lib/src/gui/qfont.rs

```diff
@@ -89,10 +91,28 @@ mod ffi {
         #[rust_name = "default_family"]
         fn defaultFamily(self: &QFont) -> QString;
 
+        /// Returns true if a window system font exactly matching
+        /// the settings of this font is available.
+        #[rust_name = "exact_match"]
+        fn exactMatch(self: &QFont) -> bool;
+
+        /// Returns the requested font family name, i.e. the name
+        /// set in the constructor or the last setFont() call.
+        fn family(self: &QFont) -> QString;
+
+        /// Returns the requested font family names, i.e. the names set in the last setFamilies()
+        /// call or via the constructor. Otherwise it returns an empty list.
+        fn families(self: &QFont) -> QStringList;
+
         /// Returns true if fixed pitch has been set; otherwise returns false.
         #[rust_name = "fixed_pitch"]
         fn fixedPitch(self: &QFont) -> bool;
 
+        /// Sets this font to match the description descrip. The description is a comma-separated
+        /// list of the font attributes, as returned by toString().
+        #[rust_name = "from_string"]
+        fn fromString(self: &mut QFont, descrip: &QString) -> bool;
```

Comment:
I wonder if we should make this return a `Result` of some sort?

Don't know what the error type would be though...

Reply:
I think it return false when we can't create font from string.
I don't see if it's necessary to return a Result here.

Reply:
`Result` just communicates better that this is an operation that may fail.

The bool indicates this as well, but it can just be ignored. A `Result` would be `#[must_use]`.
But I don't know of a good type to use, `Result<(),()>` I guess, as we don't have more information...
Would be a bit silly though I guess.
We can also merge as-is.